src/common/signalk/SKNMEAVisitor.cpp

```diff
@@ -51,6 +117,17 @@ void SKNMEAVisitor::visitSKEnvironmentOutsidePressure(const SKUpdate& u, const S
   sb.setField(3, "B");
   sb.setField(4, "Barometer");
   _sentences.add(sb.toNMEA() + "\r\n");
+  //Serial.printf("%s\n", sb.toNMEA());
+}
+
+void SKNMEAVisitor::visitSKElectricalBatteriesVoltage(const SKUpdate& u, const SKPath &p, const SKValue &v) {
```

Comment:
Have you tested this with some tools? (I am just curious - any output is better than no output).

Reply:
I could not get the BMP280 working here in I2C mode :-( so I do not know anything about pressure.
Voltage is quite ok. But could get an offset due to different electronics. 
Or just different settings in board.h